pandas/io/parquet.py

```diff
@@ -452,7 +457,9 @@ def to_parquet(
         .. versionadded:: 2.1.0
 
     kwargs
-        Additional keyword arguments passed to the engine
+        Additional keyword arguments passed to the engine.
+        When using the ``'pyarrow'`` engine ``to_pandas_kwargs`` can be used to pass
+        through arguments to ``pyarrow.Table.to_pandas``.
```

Comment:
I think this is the docstring of `to_parquet` instead of `read_parquet` ? 
(also would add an actual entry in the list of parameters instead of only mentioning it under the `kwargs` entry

Reply:
oh oops, good catch! moved the docstring to the right place and updated to make it an explicit parameter.